### PR TITLE
C-Assignment

### PR DESCRIPTION
**Pull request by destroyer2908**

### CODE REVIEW

File: Week 1/02/Q10_02_Bejohn.c [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+{
+	
+int num,i,count=0;
+	
+printf("Enter a number:");
+	
+scanf("%d",&num);
+	
+for(i=2;i<num;i++)
```
Comment (peace-shillong) [inline Week 1/02/Q10_02_Bejohn.c:17]: for loop can be improved

File: Week 1/02/Q2_02_Bejohn.c [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+#include<stdio.h>
+#include<conio.h>
+void main()
+{
+ int n,rev=0,rem;
+ clrscr();
+ printf("\n\t Enter any Number:");
+ scanf("%d",&n);
+ while(n>0)
```
Comment (peace-shillong) [inline Week 1/02/Q2_02_Bejohn.c:14]: Display error message if -ve no is entered

File: Week 1/02/Q7_02_Bejohn.c [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+int add(int a, int b)         
+    
+{
+        
+int result;
+       
+result = a+b;
+        
+return result;  
+getch();
```
Comment (peace-shillong) [inline Week 1/02/Q7_02_Bejohn.c:36]: getch() not required here

**Review by peace-shillong — COMMENTED** [review]
Indent your programs nicely to we can read them easily.
Check out all other comments also

**Comment by bejohn97** [comment]
okay Sir

**Pull request merged**